R/kmPlotWrapper.R

```diff
@@ -100,7 +99,7 @@ kmPlotWrapper <- function(x,
     on.exit(par(list(plt = plt.old, mar=mar.old)))
   }
   
-  .sibylKMplot(x, main=main, xlab=xlab, ylab=ylab, lty=lty, col=col, mark=pch, cex=cex, 
```

Comment:
@tomsydney did you test whether the survival curve are truly identical as the one produced with .sibylKMplot?

Reply:
@bobthecat We have updated the KM plot (w.r.t. drawing censor-events) so that it should mimic the behaviour in SAS.